README.md

```diff
@@ -58,6 +58,14 @@ Within `LidDrivenCavity3D/workspace` should now have appeared a multitude of dir
 
 also runs all defined operations in the appropriate order.
 
+## Environmental variables
+
+OBR workflows often rely on environmental variables to adapt a workflow to specific node or cluster properties. In workflow.yaml files for example `${{env.HOST}}` is replaced by
+`$HOST`. Additionally, `OBR_RUN_CMD` defines the exact call to for parallel runs and `OBR_PREFLIGHT` can call a script to verify your environment just before the solver execution.
```

Comment:
"[...] the exact call to for [...]" 
To for?

Reply:
good catch